AsyncDisplayKit/Details/_ASDisplayViewAccessiblity.mm

```diff
@@ -14,6 +14,27 @@
 
 #pragma mark - UIAccessibilityElement
 
+// Sort accessiblity elements by x and y origin. First sort the elements by y and than by x value.
+static NSArray *SortAccessibilityElements(NSArray *accessibleElements) {
```

Comment:
I really liked the functional style (and prefer that in general) but it is adding some overhead so we pass in a `NSMutableArray` in all of the collect as well as the sort function.

Furthermore the comparator is static now